rxmagneto/src/main/java/com/aritraroy/rxmagneto/RxMagnetoInternal.java

```diff
@@ -214,4 +225,83 @@ public void call(Emitter<ArrayList<String>> emitter) {
             }
         }, Emitter.BackpressureMode.LATEST);
     }
+
+    /**
+     * Gets the app category from the Play Store page of the given package
+     *
+     * @param context     The application context
+     * @param packageName The package name of the application
+     * @return category + categoryId
+     */
+    static Observable<Pair<String, String>> getPlayStoreAppCategory(final Context context, final String packageName, final String tag) {
+        return Observable.fromEmitter(emitter -> {
+				try {
+					 if (Connectivity.isConnected(context)) {
+
+/*
```

Comment:
These comments should be removed. A code clean-up is necessary before raising a PR.